up-linux-streamer/src/main.rs

```diff
@@ -24,6 +27,9 @@ struct StreamerArgs {
 async fn main() -> Result<(), UStatus> {
     env_logger::init();
 
+    let subscription_path = "static-configs/testdata.json".to_string();
```

Comment:
We should add this to the config.rs and update the default config so that we can modify the static subscriber cache without modifying the code.

Reply:
did this